xml/http_servers.xml

```diff
@@ -2288,6 +2288,13 @@
     <param pos="0" name="service.vendor" value="Amazon"/>
     <param pos="0" name="service.product" value="Snowball"/>
   </fingerprint>
+  <fingerprint pattern="^awselb/([\d.rc]+)$">
+    <description>Amazon Elastic Load Balancing</description>
+    <example service.version="2.0">awselb/2.0</example>
+    <param pos="0" name="service.vendor" value="Amazon"/>
+    <param pos="0" name="service.product" value="Elastic Load Balancing"/>
```

Comment:
Is it Balancing or Balancer?  Balancing is the operation it performs, Balancer is its name.

Reply:
Thinking this through, the `family` should be `Elastic Load Balancing` and product could be one of `Application Load Balancer`, `Network Load Balancer`, or `Classic Load Balancer`, from https://aws.amazon.com/elasticloadbalancing/